src/frontend-samples/zoom-to-elements-sample/ZoomToElementsApp.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import "../../common/samples-common.scss";
+import { IModelApp, IModelConnection, MarginPercent, ViewChangeOptions, Viewport, ZoomToOptions } from "@bentley/imodeljs-frontend";
+import ZoomToElementsUI, { ZoomToState } from "./ZoomToElementsUI";
+
+class ZoomToElementsAPI {
```

Comment:
This API class seems pointless.  I suggest lumping the one method into the APP class.